content/Software and Downloads/IDE Settings/Run-arduino-debug-exe-Windows.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Run arduino_debug.exe (Windows)"
+---
+
+The Java-based version of Arduino IDE (1.x) for Windows includes a debug tool for troubleshooting.
+
+---
+
+## Running `arduino_debug.exe`
+
+1. Open File Explorer and navigate to the Arduino IDE application folder, located in `C:\Program Files (x86)\Arduino` by default.
+
+2. Find the `arduino_debug.exe` file and double-click it to launch it.
```

Comment:
By far, the most common use for the output provided by `arduino_debug.exe` is troubleshooting the IDE failing to start. If the user starts the executable by double-clicking, they will see the terminal window open for a fraction of a second and then close before they were able to look at the valuable output.

In this case, the user must run `arduino_debug.exe` from a command line terminal (PowerShell or cmd).

The IDE generally does a good job of providing information about the problem via the application when it is able to start. The output you would see from `arduino_debug.exe` while it is running is also printed to the output panel at the bottom of the Arduino IDE window.

For this reason, I think you should drop this idea of telling people to open it via a double-click and the tutorial should instruct the user in how to run it from the command line terminal.

Reply:
Great feedback, I will make that change.

Reply:
e016fab12a76f58fa26001f57ca1e47bbdbbc68f